.gitignore

```diff
@@ -3,3 +3,7 @@ dist
 pynsq.egg-info
 build
 docs/_build
+.cache
+.eggs
+*.dat
+*.tmp
```

Comment:
Yep, indeed, but running tests produced too much crap. Not sure if this could make a decent PR on its own.